pom.xml

```diff
@@ -91,9 +91,10 @@
         <hbase.guava.version>12.0.1</hbase.guava.version>
         <cassandra.version>3.0.0</cassandra.version>
         <cassandra.guava.version>18.0</cassandra.guava.version>
-        <kafka.08.version>0.8.2.1</kafka.08.version>
-        <kafka.09.version>0.9.0.1</kafka.09.version>
-        <kafka.10.version>0.10.0.1</kafka.10.version>
+        <kafka.version>0.9.0.1</kafka.version> <!-- also supports: 0.10.2.1, 1.0.0 -->
+        <zkclient.version>0.7</zkclient.version> <!-- corresponding versions: 0.10, 0.10 -->
+        <kafka.jopt.version>3.2</kafka.jopt.version> <!-- corresponding versions: 4.9, 5.0.4 -->
```

Comment:
Should we document how developers can manage all the versions?  I *think* I'd know what to do, but I doubt that it is clear to an outside developer.

Reply:
you shouldn't actually need to change the versions, as it's a provided dep you plug in at runtime. but looks like the kakfa datastore docs need some updating, so I'll add it in there as part of the big docs pr